### ISSUE TITLE
Load pretrained model size mismatch.

### ISSUE THREAD

**Issue by bimo-adiparwa** [Monu-Khicher-1/multi-stage-learning#2]
Hi, when I tried to run test. I got this error.
```bash
Unexpected key(s) in state_dict: "backbone.patch_embed.backbone.layers.3.downsample.norm.weight", "backbone.patch_embed.backbone.layers.3.downsample.norm.bias", "backbone.patch_embed.backbone.layers.3.downsample.reduction.weight", "backbone.patch_embed.backbone.head.fc.weight", "backbone.patch_embed.backbone.head.fc.bias", "embedder.layers.3.downsample.norm.weight", "embedder.layers.3.downsample.norm.bias", "embedder.layers.3.downsample.reduction.weight", "embedder.head.fc.weight", "embedder.head.fc.bias". 
        size mismatch for backbone.patch_embed.backbone.layers.1.downsample.norm.weight: copying a param with shape torch.Size([384]) from checkpoint, the shape in current model is torch.Size([768]).
        size mismatch for backbone.patch_embed.backbone.layers.1.downsample.norm.bias: copying a param with shape torch.Size([384]) from checkpoint, the shape in current model is torch.Size([768]).
        size mismatch for backbone.patch_embed.backbone.layers.1.downsample.reduction.weight: copying a param with shape torch.Size([192, 384]) from checkpoint, the shape in current model is torch.Size([384, 768]).
        size mismatch for backbone.patch_embed.backbone.layers.2.downsample.norm.weight: copying a param with shape torch.Size([768]) from checkpoint, the shape in current model is torch.Size([1536]).
        size mismatch for backbone.patch_embed.backbone.layers.2.downsample.norm.bias: copying a param with shape torch.Size([768]) from checkpoint, the shape in current model is torch.Size([1536]).
        size mismatch for backbone.patch_embed.backbone.layers.2.downsample.reduction.weight: copying a param with shape torch.Size([384, 768]) from checkpoint, the shape in current model is torch.Size([768, 1536]).
        size mismatch for embedder.layers.1.downsample.norm.weight: copying a param with shape torch.Size([384]) from checkpoint, the shape in current model is torch.Size([768]).
        size mismatch for embedder.layers.1.downsample.norm.bias: copying a param with shape torch.Size([384]) from checkpoint, the shape in current model is torch.Size([768]).
        size mismatch for embedder.layers.1.downsample.reduction.weight: copying a param with shape torch.Size([192, 384]) from checkpoint, the shape in current model is torch.Size([384, 768]).
        size mismatch for embedder.layers.2.downsample.norm.weight: copying a param with shape torch.Size([768]) from checkpoint, the shape in current model is torch.Size([1536]).
        size mismatch for embedder.layers.2.downsample.norm.bias: copying a param with shape torch.Size([768]) from checkpoint, the shape in current model is torch.Size([1536]).
        size mismatch for embedder.layers.2.downsample.reduction.weight: copying a param with shape torch.Size([384, 768]) from checkpoint, the shape in current model is torch.Size([768, 1536]).
```

Is it different pretrained model?

**Comment by Monu-Khicher-1** [comment]
@bimo-adiparwa 
Can you give some more details? Like image size and the command you run. It will help me to understand the problem.

**Comment by PYM-242115005** [comment]
Hi,I`m a student who wants to reproduce the experiment.After reading the file README,I wonder if did it wrong.I just built a virtual  environment satisfying the need,and cut the Celeb-V2 vedios into images 1fps,cropped all images,then masked the training data. After this,I started the three-stage training as the README said.But the result is definitely wrong.I actually wonder what should I adjust more as a fresh one to reproduce it.By the way,I found a minority of images were not masked when I tried to do so.I just deleted them.I`m so confused now,and I really need a instruction to get a result as excellent as yours based on the codes you provided on github! Hoping for your reply.Thanks!!!
